packages/blade/src/components/SideNav/_decisions/decisions.md

```diff
@@ -0,0 +1,418 @@
+
+#### Example Nav Links
+
+<table>
+
+<tr><td>Code</td><td>Preview</td></tr>
+
+<tr>
+<td>
+
+<!-- prettier-ignore -->
+```jsx
+<SideNavLink
+  icon={BillIcon}
+  title="Vendor Payments"
+  href="/vendor-payments"
+/>
+```
+
+</td>
+<td><img src="navlink-eg-1.png" width="200px" /></td>
+</tr>
+
+<tr>
+<td>
+
+<!-- prettier-ignore -->
+```jsx
+<SideNavLink
+  icon={PayoutIcon}
+  title="Create Payouts"
+  href="/payouts"
+  trailing={
+    <Button
+      href="/payouts/create"
+      icon={PlusIcon}
+      size="xsmall"
+      variant="tertiary"
+    />
+  }
+/>
+```
+
+</td>
+<td><img src="navlink-eg-2.png" width="200px" /></td>
+</tr>
+
+<tr>
+<td>
+
+<!-- prettier-ignore -->
+```jsx
+<SideNavLink
+  icon={LayoutIcon}
+  title="L1 Item Name"
+  href="/new-item-link"
+  titleSuffix={
+    <Badge color="positive">New</Badge>
+  }
+/>
+```
+
+</td>
+<td><img src="navlink-eg-3.png" width="200px" /></td>
+</tr>
+
+<tr>
+
+<td>
+
+<!-- prettier-ignore -->
+```jsx
+<Tooltip content="Action Name (Cmd + P)">
+  <SideNavLink
+    icon={LayoutIcon}
+    title="L1 Item Name"
+    href="/new-item-link"
+  />
+</Tooltip>
+```
+
+</td>
+<td><img src="navlink-eg-4.png" width="240px" /></td>
+</tr>
+
+</table>
+
+### SideNavLevel
+
+Nested SideNavLevel components create new levels. This can be used to create L1 - L2 - L3 levels in your navbar
+
+| **Props** | **Description**                                       | **Type** | **Default Value** |
+| --------- | ----------------------------------------------------- | -------- | ----------------- |
+| title     | Only applicable in L2. Becomes the title of the Level | string   |                   |
+| children  | children slot. Accepts SideNavLink as children        | JSX      |                   |
+
+<table>
+<tr>
+<td>
+
+```jsx
+<SideNav>
+  {/* L1 Items */}
+  <SideNavLink title="L1 Item" />
+
+  <SideNavLink title="L2 Trigger">
+    {/* L2 Level */}
+    <SideNavLevel title="L2 Menu Heading">
+      <SideNavLink />
+      <SideNavLink />
+      <SideNavLink title="L3 Trigger">
+        {/* L3 Level */}
+        <SideNavLevel>
+          <SideNavLink />
+          <SideNavLink />
+        </SideNavLevel>
+      </SideNavLink>
+    </SideNavLevel>
+</SideNav>
+```
+
+</td>
+
+<td>
+<img width="500px" src="navlevels-eg.png" />
+</td>
+</tr>
+</table>
+
+### SideNavSection
+
+| **Props**       | **Description**                                              | **Type** | **Default Value** |
+| --------------- | ------------------------------------------------------------ | -------- | ----------------- |
+| title           | title of the section                                         | string   |                   |
+| maxVisibleItems | Number of items visible (rest go inside +x more collapsible) | number   | undefined         |
+| children        | Children slot. For SideNavLink children items                | JSX      |                   |
```

Comment:
when a user clicks the +13 more we need to give them a trigger to track analytics

Reply:
yup right. Will add this